docs/UserGuide.adoc

```diff
@@ -158,21 +180,45 @@ Selects the 2nd person in the address book.
 `select 1` +
 Selects the 1st person in the results of the `find` command.
 
-=== Listing entered commands : `history`
+=== Listing entered commands : `history` or `hist`
 
 Lists all the commands that you have entered in reverse chronological order. +
-Format: `history`
+Format: `history` or `hist`
 
 [NOTE]
 ====
 Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
 ====
 
+=== Email appointment reminder  : `emailreminder` or `er` (tentative)
```

Comment:
Can include more such things in UG, which you haven't implemented yet(mark them as coming in 2.0). You may decide later if you want to implement those features or not